Silicon/QemuSocPkg/FspBin/Patches/0001-Build-QEMU-FSP-2.0-binaries.patch

```diff
@@ -1479,6 +1526,23 @@ index 0000000000..6769d7f53d
 +    );
 +  }
 +
++  // Report TSEG memroy
++  BuildResourceDescriptorWithOwnerHob (
++      EFI_RESOURCE_MEMORY_RESERVED,
++      (
++        EFI_RESOURCE_ATTRIBUTE_PRESENT |
++        EFI_RESOURCE_ATTRIBUTE_INITIALIZED |
++        EFI_RESOURCE_ATTRIBUTE_TESTED |
++        EFI_RESOURCE_ATTRIBUTE_UNCACHEABLE |
++        EFI_RESOURCE_ATTRIBUTE_WRITE_COMBINEABLE |
++        EFI_RESOURCE_ATTRIBUTE_WRITE_THROUGH_CACHEABLE |
++        EFI_RESOURCE_ATTRIBUTE_WRITE_BACK_CACHEABLE
++      ),
++      LowMemLen,
```

Comment:
Can we declare/use a TsegBase variable instead of using LowMemLen? At first glance, it looks a size, not a base.

Reply:
Make sense.  I will make the change.

Reply:
Committed new changes per the review comments.